### PR TITLE
added concepts

### PR DESCRIPTION
**Pull request by Priyansi**
Shift concepts from the old website.

### CODE REVIEW

**Review by ydcjeff — DISMISSED** [review]

**Comment by vfdev-5** [comment]
@Priyansi let's also add a link on Concepts from https://deploy-preview-135--pytorch-ignite.netlify.app/getting-started/

File: src/concepts/_index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Concepts
+weight: 3
+---
+
+# Concepts
+
+Understand the essence of the library in detail and why things are built a certain way.
```
Comment (vfdev-5) [inline src/concepts/_index.md:8]:
```suggestion
Understand the essence of the library in detail and why things are built in this way.
```

File: src/concepts/02-events-and-handlers.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    event_name=Events.EPOCH_COMPLETED,
+    log_handler=WeightsHistHandler(model)
+)
+```
+
+## Timeline and events
+
+Below the events and some typical handlers are displayed on a timeline for a training loop with evaluation after every epoch:
+
+![png](/_images/timeline_and_events.png)
```
Comment (vfdev-5) [inline src/concepts/02-events-and-handlers.md:204]: Let's rework this image, https://github.com/pytorch/ignite/issues/2282
In dark mode it is not readable

**Review by ydcjeff — APPROVED** [review]

**Review by vfdev-5 — APPROVED** [review]
LGMT! Let's merge it and update the events/handers image in another PR

**Pull request merged**